sycl/test/basic_tests/event.cpp

```diff
@@ -1,5 +1,3 @@
-// REQUIRES: opencl || level0
-
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -L %opencl_libs_dir -lOpenCL
```

Comment:
You can remove -lOpenCL because thre eare no OpenCL direct calls in the source anymore.